torch/_inductor/codecache.py

```diff
@@ -1519,6 +1523,17 @@ def set(cls, key: str, params: dict[str, str], cubin: str, bin_type: str) -> Non
                 config.aot_inductor.output_path
             )[0],
         )
+        if config.aot_inductor.package_cpp_only:
+            assert config.triton.unique_kernel_names, (
+                "package_cpp_only requires triton kernel names to be unique"
+            )
+            dir_name = os.path.dirname(path)
+            _, ext = os.path.splitext(path)
+            # Construct the new full path
+            new_path = os.path.join(dir_name, params["mangled_name"] + ext)
```

Comment:
what do the names look like now?

Reply:
It's something like `triton_poi_fused_add_addmm_0.cubin`